# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/mtransport/ipc/WebrtcTCPSocket.h
# Commit: b83124cf3a89
# Full Hash: b83124cf3a891091773bcfe2cd15e36155fc5a2d
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2019-09-18 21:49:27
# Regressor Bug: 1569183
# File Overlap Count: 2
# Description:
#   Bug 1569183: Teach WebrtcTCPSocket to use TLS when configured to. r=mjf,mayhemer
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D45103
# ==============================================================================

diff -r b879a0044cf3 -r b83124cf3a89 media/mtransport/ipc/WebrtcTCPSocket.h
--- a/media/mtransport/ipc/WebrtcTCPSocket.h	Tue Sep 17 18:17:18 2019 +0000
+++ b/media/mtransport/ipc/WebrtcTCPSocket.h	Tue Sep 17 18:17:24 2019 +0000
@@ -54,7 +54,8 @@
   void SetTabId(dom::TabId aTabId);
   nsresult Open(const nsCString& aHost, const int& aPort,
                 const nsCString& aLocalAddress, const int& aLocalPort,
-                const net::LoadInfoArgs& aArgs, const nsCString& aAlpn,
+                bool aUseTls, const net::LoadInfoArgs& aArgs,
+                const nsCString& aAlpn,
                 NrSocketProxyConfig::ProxyPolicy aProxyPolicy);
   nsresult Write(nsTArray<uint8_t>&& aBytes);
   nsresult Close();
@@ -77,7 +78,7 @@
   nsCOMPtr<nsIURI> mURI;
   net::LoadInfoArgs mLoadInfoArgs;
   nsCString mAlpn;
-  bool mSsl = false;
+  bool mTls = false;
   bool mForceProxy = false;
   nsCString mLocalAddress;
   uint16_t mLocalPort = 0;